CMakeLists_files.cmake

```diff
@@ -76,12 +76,12 @@ endif()
 # originally generated with the command:
 # find tests/not-unit/ -name \*.cpp -o \*.cc
 list (APPEND ATTIC_FILES
-  attic/partition_test.cpp
+#  attic/partition_test.cpp
 # attic/dumux_test.cpp
-  attic/mapper_test.cpp
-  attic/buildcpgrid_test.cpp
+#  attic/mapper_test.cpp
+#  attic/buildcpgrid_test.cpp
 # attic/cpgrid_test.cpp
-  attic/check_grid_normals.cpp
+#  attic/check_grid_normals.cpp
   )
```

Comment:
Maybe we could just remove this list altogether?

Reply:
we can.